package.json

```diff
@@ -20,7 +20,7 @@
     "mocha": "6.1.4",
```

Comment:
- Please update the changelog under the [unreleased] section
- Might want to expose the LockFactory in the `/utils/lockFactory` folder similar to how the [windowHandler is exposed](https://github.com/sublimator/supertokens-website/blob/nd-add-lock-factory-to-config-2023-01-13/utils/windowHandler/index.js)

Reply:
The other changes look good. So once these two comments are addressed, i'll merge and release this.